daemon/provider/musicvideoinfo/spotifyprovider.py

```diff
@@ -173,3 +177,54 @@ def get_tracks_by_album_id(self, album_id):
                 break
 
         return tracks
+
+    def get_new_release_by_album_id(self):
+        self.logger.info("Get albums of the new release from Spotify.")
+        service_host = "https://api.spotify.com"
+        """
+            Get client_id and client_secret value from local.
+        """
+        config = Config("../../xmusic.cfg")
+        print("1")
+        client_id = config.spotify_client_id
+        client_secret = config.spotify_client_secret
+
+        access_token = Spotify(client_id, client_secret).get_token()
+        headers = ["Authorization: Bearer " + access_token]
+
+        offset = 0
+        limit = 20
+        new_release = []
+        while True:
+            params = {"offset": offset*limit, "limit": limit}
+            buf = BytesIO()
+            client = pycurl.Curl()
+            client.setopt(pycurl.HEADER, False)
+            client.setopt(pycurl.HTTPHEADER, headers)
+            client.setopt(pycurl.URL, service_host + "/v1/browse/new-releases" + "?" + urlencode(params))
+            client.setopt(pycurl.WRITEFUNCTION, buf.write)
+            client.perform()
+            client.close()
+            body = json.loads(buf.getvalue().decode("utf-8"))
+            buf.close()
+
+            for album_data in body["albums"]["items"]:
+                image = []
+                for image_data in album_data["images"]:
+                    image.append((image_data["url"],
+                                  image_data["width"],
+                                  image_data["height"]))
+
+                artists = album_data["artists"][0]
+                artist_data = (artists["name"], artists["id"])
+                new_release.append((album_data["id"],
+                                    album_data["type"],
+                                    artist_data,
+                                    image))
+
+            if body["albums"]["total"] > body["albums"]["offset"]:
+                offset = offset + 1
+            else:
+                break
+
+        return new_release
```

Comment:
It's much better to get `Album[]` which is defined in the `entity.py` as the result.